presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/PreferredProperties.java

```diff
@@ -130,8 +132,12 @@ public static PreferredProperties local(List<? extends LocalProperty<VariableRef
         return localProperties;
     }
 
-    public PreferredProperties mergeWithParent(PreferredProperties parent)
+    public PreferredProperties mergeWithParent(PreferredProperties parent, Session session)
```

Comment:
This change might be conflicting with https://github.com/prestodb/presto/pull/11262/files#diff-ab30f9bc656641796bf023d3139c5ec7R133